File: Python/D10 - T/2 - series/2.py
# Write a program to find the sum of following series:
# 1 + (1/3) + (1/5) + (1/7) + ....

n = int(input('Enter number : '))
sum = 0

for i in range(1, n+1, 2):
    print(i)
    sum = sum + (1/i)

print(sum)

File: C/D6/2/2.cpp
// Write a C program to input an integer number and check the
// last digit of number is even or odd.

#include <stdio.h>

int main(){
    int num, last_digit;

    printf("Enter a number : ");
    scanf("%d", &num);

    last_digit = num % 10;
    printf("%d", last_digit);
    
    (last_digit % 2 == 0)
            ? printf("Last digit of entered number is even")
            : printf("Last digit of entered number is odd");
    
}